libsolidity/analysis/SyntaxChecker.cpp

```diff
@@ -269,6 +269,24 @@ bool SyntaxChecker::visit(VariableDeclaration const& _declaration)
 	return true;
 }
 
+bool SyntaxChecker::visit(VariableDeclarationStatement const& _statement)
+{
+	// Check if declaration contains at least one named component, and report otherwise.
+	size_t const namedVariablesCount = count_if(
```

Comment:
Can you use `boost::range::all_of(_statement.declarations(), nullptr)`?

Reply:
@chriseth uff, I naturally try to reduce the amount of boost in my code, but yes, of course. :-)

Reply:
``std::all_of`` :)?

Reply:
That would mean back to iterators and a predicate, though.